homeassistant/components/airgradient/coordinator.py

```diff
@@ -47,16 +46,12 @@ async def _update_data(self) -> _DataT:
 class AirGradientMeasurementCoordinator(AirGradientCoordinator[Measures]):
     """Class to manage fetching AirGradient data."""
 
-    _update_interval = timedelta(minutes=1)
-
     async def _update_data(self) -> Measures:
         return await self.client.get_current_measures()
 
 
 class AirGradientConfigCoordinator(AirGradientCoordinator[Config]):
     """Class to manage fetching AirGradient data."""
 
-    _update_interval = timedelta(minutes=5)
```

Comment:
It would be nice to explain why its moving from 5 minutes to 1 minute here in the opening text

Reply:
Updated, thanks